recipes/bioconductor-mirnatap.db/meta.yaml

```diff
@@ -0,0 +1,32 @@
+package:
+  name: bioconductor-mirnatap.db
+  version: 0.99.10
+source:
+  fn: miRNAtap.db_0.99.10.tar.gz
+  url: https://bioconductor.org/packages/release/data/annotation/src/contrib/miRNAtap.db_0.99.10.tar.gz
+  md5: 48010280b68d6f1ddbff3b374132ce0a
+build:
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+requirements:
+  build:
+    - bioconductor-mirnatap
+    - bioconductor-annotationdbi
+    - r-base
+    - r-dbi
+    - r-rsqlite
```

Comment:
Since `bioconductor-mirnatap` [will] included all those dependencies, you don't them.
Did you use this script to build your packages: [bioconductor_skeleton.py]( https://bioconda.github.io/guidelines.html#r-bioconductor)?

But please wait to the validation of the package bioconductor-mirnatap before push an other version.